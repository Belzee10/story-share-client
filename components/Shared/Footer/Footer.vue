<template>
  <v-footer class="d-flex justify-center">
    <div class="py-2">
      <p class="ma-0">
        Made with love by
        <a href="#">@belzee5</a>
      </p>
    </div>
  </v-footer>
</template>

<script>
export default {
  name: 'Footer'
};
</script>

<style lang="scss" scoped></style>
