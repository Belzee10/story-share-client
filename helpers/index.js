/**
 * transform the response from express
 * remove values from exclude
 * change the key "_id" by "id"
 * @param {Object} item
 * @param {Array} exclude
 */
export const transformResponse = (item, exclude = ['__v', 'created_at']) => {
  Object.keys(item).forEach(key => {
    if (exclude.includes(key)) delete item[key];
  });
  if (item._id) {
    item.id = item._id;
    delete item._id;
  }
  return item;
};
