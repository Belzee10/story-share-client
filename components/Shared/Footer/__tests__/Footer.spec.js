import Footer from '../Footer.vue';
import { createWrapper } from '~/config/tests.helpers';

describe('Footer.vue', () => {
  test('should render correctly', () => {
    const wrapper = createWrapper(Footer);
    expect(wrapper.element).toMatchSnapshot();
  });
});
