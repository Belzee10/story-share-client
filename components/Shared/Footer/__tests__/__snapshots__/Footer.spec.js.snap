// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue should render correctly 1`] = `
<footer
  class="v-footer d-flex justify-center v-sheet v-sheet--tile theme--light"
>
  <div
    class="py-2"
  >
    <p
      class="ma-0"
    >
      
      Made with love by
      
      <a
        href="#"
      >
        @belzee5
      </a>
    </p>
  </div>
</footer>
`;
